projects/plugins/transformers/blob-download-plugin/pom.xml

```diff
@@ -137,5 +137,9 @@
 			<artifactId>javax.json</artifactId>
 			<version>1.0.4</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>javax.activation</groupId>
+            <artifactId>activation</artifactId>
+        </dependency>
```

Comment:
I don't think this dependency is necessary here. Did you really test the blob-download-plugin?